pkg/endpoint/endpoint.go

```diff
@@ -2739,3 +2630,66 @@ func (e *Endpoint) scrubIPsInConntrackTable() {
 	e.scrubIPsInConntrackTableLocked()
 	e.Unlock()
 }
+
+// GetUUID returns the UUID of the endpoint
+func (e *Endpoint) GetUUID() string {
+	return e.UUID
```

Comment:
Why have a getter for UUID if the field is public?

Reply:
Removed

Reply:
EDIT: I forgot that this was added to implement the `Buildable` interface. I've added it back and added a comment to mention the interface use.